web-api/src/storage/postgres.rs

```diff
@@ -165,10 +197,10 @@ impl Database {
                 builder
                     .reset()
                     .push_tuple(ids.by_ref().take(Self::BIND_LIMIT))
-                    .push(" RETURNING document_id, is_candidate;")
+                    .push(" RETURNING document_id, is_candidate, coalesce(array_length(embeddings, 1), 0) as embedding_count;")
```

Comment:
:art: in the [postgres docs this function name is capital case](https://www.postgresql.org/docs/15/functions-conditional.html#FUNCTIONS-COALESCE-NVL-IFNULL). similarly below.
```suggestion
                    .push(" RETURNING document_id, is_candidate, COALESCE(array_length(embeddings, 1), 0) as embedding_count;")
```